### ISSUE TITLE
"Recorded" message displayed twice when stopping

### ISSUE THREAD

**Issue by real-or-random** [GothenburgBitFactory/timewarrior#116]
```> timew
There is no active time tracking.
> timew start test
Tracking test
  Started 2018-03-09T23:56:16
  Current                  16
  Total               0:00:00
> timew stop
Recorded test
  Started 2018-03-09T23:56:16
  Ended                    19
  Total               0:00:03
Recorded test
  Started 2018-03-09T23:56:16
  Ended                    19
  Total               0:00:03
> timew
There is no active time tracking.
> timew --version                                                                                                                                                              
1.1.1
```

This happens with every tag... It started a few ago... I can't really remember, and I never really bothered, because the intervals are recorded correctly, i.e., only once. Just the message appears twice.

At first glance I can't find anything suspicious in the data files.

**Comment by real-or-random** [comment]
The config file is empty by the way.

**Comment by real-or-random** [comment]
Just verified that this happens even with an empty database. (This makes me wonder that I'm the first to report this bug. Or is there anything special about my system?)

**Comment by lauft** [comment]
I confirmed your bug and fixed it with 5264947de8f23ae5cfcb338bd7dbaeb0dab797fa.

> Just verified that this happens even with an empty database.
Can you maybe elaborate on that and provide an example?
